whatis277/harvest
# bean: Add passwordless auth email sending

Sends the passwordless auth email which contains a login URL

Testing instructions:
1. [`localhost:8080/get-started`](http://localhost:8080/get-started)
2. Enter an email address 
3. Check [`localhost:8025`](http://localhost:8025)
4. Ensure you see an email with a login link

<img width="1062" alt="image" src="https://github.com/whatis277/harvest/assets/5631091/57582708-6cb0-402b-85eb-05ef9763d6e8">
